main.ts

```diff
@@ -59,8 +57,8 @@ function createWindow() {
 
   nativeTheme.themeSource = 'light';
 
-  win.setTitle(require('./package.json').name + ' ' + require('./package.json').version);
-  if (serve) {
+  win.setTitle(appTitle);
```

Comment:
Also just being a bit nitpicky here, not a big deal but maybe  we can shift this line to above the `nativeTheme.themeSource = 'light';` and place it directly under the instantiation of the `win` variable without a spacing gap between the `setTitle` and the instantiation?

so it'll be something like

```typescript
// Create the browser window.
win = new BrowserWindow(windowOptions);
win.setTitle(appTitle);

nativeTheme.themeSource = 'light';

...

```

In the future if we are ever looking for any manipulation regarding the `win` variable (which in this case might be unlikely with the web-migration and all but still) at least it'll be clumped together.